demos/browser/app/meeting/CloudWatchLogger.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import LogLevel from '../../../../src/logger/LogLevel';
+import {ConsoleLogger} from '../../../../src/index';
+
+export default class CloudWatchLogger extends ConsoleLogger {
+  name: string;
+  level: LogLevel;
+  logCapture: string[] = [];
+  meetingID: string;
+  attendeeID: string;
+  logSeqNo: any;
+  lock = false;
+  batchSizes = 50;
+
+  constructor(name: string, level = LogLevel.WARN, meetingID: string, attendeeID: string) {
+    super(name, level);
+    this.name = name;
+    this.level = level;
+    this.meetingID = meetingID;
+    this.attendeeID = attendeeID;
+    this.logSeqNo = 0;
+  }
+
+  async publishToCloudWatch(base_url: string) {
+    var _this = this;
```

Comment:
I understand that we have used this pattern in the demo app, but a better pattern is to use an arrow function.

1. Remove this line.
2. Use an arrow function and replace ```_this``` with ```this```.
  ```
  setInterval(async () => {
     // In an arrow function, the ```this``` context is the same as the outer scope.
     if (this.lock == true || this.logCapture.length == 0 )
     ...
  }, 5000);
  ```